pyproject.toml

```
@@ -0,0 +1,111 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-dax-ppdb"
+description = "Prompt Products Database for LSST AP pipeline."
+license = {text = "GPLv3+ License"}
+readme = "README.md"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Astronomy",
+]
+keywords = ["lsst"]
+dependencies = [
+    "astropy",
+    "numpy",
+    "pandas",
+    "pyyaml >= 5.1",
+    "sqlalchemy",
+    "felis",
```

Maybe this is your problem.
```suggestion
    "lsst-felis",
```

---

Ah, thanks! 🤦‍♂️